reqtraq_config.json

```diff
@@ -16,11 +20,23 @@
         },
         {
             "path": "certdocs/TRAQ-137-SRD.md",
-            "requirements": "REQ-TRAQ-SWH-(\\d+)"
+            "requirements": "REQ-TRAQ-SWH-(\\d+)",
+            "attributes": [
+                {
+                    "name": "Parents",
+                    "value": "REQ-TRAQ-SWH-(\\d+)"
```

Comment:
Is this right? atributes.json had REQ-TRAQ-SYS as parents.

Reply:
no, this is not correct. good catch! will fix